app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt

```diff
@@ -45,32 +47,66 @@ import kotlinx.parcelize.Parcelize
 
 @OptIn(ExperimentalMaterial3AdaptiveApi::class)
 @Composable
-fun TransfersScreenWrapper(direction: TransferDirection) {
+fun TransfersScreenWrapper(direction: TransferDirection, transferUuid: String? = null) {
     var hasTransfer: Boolean by rememberSaveable { mutableStateOf(false) }
 
     TwoPaneScaffold<DestinationContent>(
-        listPane = { ListPane(direction, navigator = this, hasTransfer = { hasTransfer = it }) },
-        detailPane = { DetailPane(navigator = this, hasTransfer) },
+        listPane = {
+            val transfersViewModel = hiltViewModel<TransfersViewModel>()
+            val isDeepLinkConsumed by transfersViewModel.isDeepLinkConsumed.collectAsStateWithLifecycle()
+            HandleDeepLink(
+                transferUuid = transferUuid,
+                isDeepLinkConsumed = { isDeepLinkConsumed },
+                consumeDeepLink = transfersViewModel::consumeDeepLink,
+                direction = direction,
+            )
+            ListPane(
+                direction,
+                navigator = this,
+                updateHasTransfer = { hasTransfer = it },
+                transfersViewModel = transfersViewModel
+            )
+        },
+        detailPane = {
+            DetailPane(navigator = this, hasTransfer = { hasTransfer })
+        },
     )
 }
 
+@OptIn(ExperimentalMaterial3AdaptiveApi::class)
+@Composable
+private fun ThreePaneScaffoldNavigator<DestinationContent>.HandleDeepLink(
```

Comment:
Since this function doesn't display any UI, we should write it with a lowercase first letter.